rotkehlchen/exchanges/manager.py

```diff
@@ -307,3 +308,13 @@ def get_user_binance_pairs(self, name: str, location: Location) -> list[str]:
     def query_history_events(self) -> None:
         for exchange in self.iterate_exchanges():
             exchange.query_history_events()
+
+    def get_exchange_mappings(self) -> LocationEventMappingType:
+        """Collect from the different exchanges the event mappings"""
+        mappings: LocationEventMappingType = {}
+        for location, exchanges in self.connected_exchanges.items():
+            for exchange in exchanges:
+                if (exchange_mapping := exchange.get_event_mappings()) != {}:
```

Comment:
nitpick: I think doing `if len(exchange_mapping := exchange.get_event_mappings()) != 0:` is a bit safer